src/common/task_manager/gc.rs

```diff
@@ -110,9 +110,11 @@ impl GcTask {
     /// Run the GC task, consume the task itself.
     #[allow(clippy::pattern_type_mismatch)] // for `tokio::select!`
     pub async fn run(mut self, token: CancellationToken) -> Self {
+        info!("GC task `{:?}` starts.", self.name);
```

Comment:
These debug log has no use, should delete

Reply:
Done.